### PR TITLE
Update to Vuetify 1.0

### PR DESCRIPTION
**Pull request by voluntadpear**
Updated to Vuetify 1.0.0, the latest version

### CODE REVIEW

File: client/components/document.vue [hunk 1]
```diff
@@ -50,26 +50,22 @@
...
-                    <v-card-text>
-                      Comment 2.
-                    </v-card-text>
-                  </v-card>
-                </v-tabs-content>
-              </v-tabs-items>
+              <v-tab ripple href="#comments" class="primary--text">Comments</v-tab>
+              <v-tab ripple href="#chat" class="primary--text"><v-badge><span slot="badge">4</span>Chat</v-badge></v-tab>
+              <v-tab ripple href="#history" class="primary--text">History</v-tab>
+              <!-- <v-tabs-slider color="primary" /> -->
```
Comment (mitar) [inline client/components/document.vue]: This is not needed anymore? So do they provide a slider if it is too narrow?

**Comment by voluntadpear** [comment]
This PR is ready for review

File: client/components/document.vue [hunk 2]
```diff
@@ -12,38 +12,38 @@
       <v-container fluid class="app-layout__users">
         <v-layout row wrap justify-start align-content-start>
           <v-flex class="app-layout__user">
-            <v-btn flat icon style="border-color: #9fa8da;">
```
Comment (mitar) [inline client/components/document.vue]: Why removing flat? I think it is not needed, but still, why decided to remove it?

**Comment by mitar** [comment]
I fixed avatar location differently. There was before 1px of empty space between the avatar and the border. You removed that by increasing the size of avatars.

**Pull request merged**